addon/plugins/rdfa-date-plugin/index.ts

```diff
@@ -11,7 +11,7 @@ export const defaultDateFormats: DateFormat[] = [
     label: 'Long Date',
     key: 'long',
     dateFormat: 'EEEE dd MMMM yyyy',
-    dateTimeFormat: 'EEEE dd MMMM yyyy at HH:mm',
+    dateTimeFormat: 'EEEE dd MMMM yyyy \'at\' HH:mm',
```

Comment:
we should use `PPPPp` instead, now there's a random `at` that's not localized

Reply:
true, sorry you told me, it's monday, the problem with PPPPp is that it doesn't give the user a start from where customize it's date, but I think is the best solution

Reply:
what do you mean with "us date"?

Reply:
Like from DD/MM/YYYY hh:mm, the user can easily change it to replace the separators, add new words...., this doesn't happen with the PPPPp but I think is also okay